samples/wifi/sta/README.rst

```diff
@@ -129,6 +135,37 @@ Testing
       <inf> sta: ==================
       <inf> sta: State: DISCONNECTED
 
+Power management testing
+************************
+
+This sample can be used to measure current consumption of both the nRF5340 SoC and nRF7002 DK independently by using two separate Power Profiler Kits (PPK).
```

Comment:
I think the correct name for the power profiler is "Power Profiler Kit II (PPK2)". I'll let someone else decide how to make plural...